content/summits/developer/2025/_index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: 2024 Developer Summit
+subtitle: |
+  The Third Scientific Python Developer Summit
+  11-15 May 2025 – Seattle, WA
+summary: |
+  The third Scientific Python Developer Summit (May 11-15, 2024) will be hosted by the eScience Institute at the University of Washington. The week-long summit will bring together forty participants, who will develop shared infrastructure for libraries in the Scientific Python ecosystem.
```

Comment:
Can probably give a better description here, given the emphasis of this year's meeting.

Reply:
I don't believe the summary currently shows up anywhere. So I am not going to worry about it for now.